addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -384,6 +386,9 @@ Composer.props = {
     hasSendButton: {
         type: Boolean,
     },
+    hasTextInputSendOnEnterEnabled: {
+        type: Boolean,
+    },
```

Comment:
shorter version: `hasTextInputSendOnEnterEnabled: Boolean,`

(don't bother changing for older prop specs, will make the change everywhere soon)